run-ci.sh

```diff
@@ -0,0 +1,20 @@
+FIRST_COMMIT_HASH=$( git log master..$( git branch --show-current ) --pretty=oneline | tail -1 | awk '{ print $1 }' )
+LAST_COMMIT_HASH=$( git rev-parse HEAD )
+ARTICLES=$( git diff --diff-filter=d --name-only ${FIRST_COMMIT_HASH}^ ${LAST_COMMIT_HASH} "wiki/**/*.md" "news/*.md" )
+
+echo -e "\n--- Binary file size check ---\n"
+bash .github/scripts/ci/inspect_binary_file_sizes.sh ${FIRST_COMMIT_HASH} ${LAST_COMMIT_HASH}
```

Comment:
since it's not GitHub-only now, we could move the scripts to their own top-level directory and blacklist it in [`.gitattributes`](https://github.com/ppy/osu-wiki/blob/master/.gitattributes)